somm_proto/Cargo.toml

```diff
@@ -11,4 +11,4 @@ prost = "0.7"
 prost-types = "0.7"
 bytes = "1"
 cosmos-sdk-proto = {git="http://github.com/cosmos/cosmos-rust", branch="main"}
-tonic = "0.4"
\ No newline at end of file
+tonic = "0.4"
```

Comment:
Not specific to this line: Is there a reason we don't add serde Deserialize/Serialize attributes during the build?

Reply:
Not sure, this bit hasn't been particularly touched in a long time.